requirements/minimum.old

```diff
@@ -8,3 +8,4 @@ timm==0.9.0
 torch==1.9.0
 torchvision==0.10.0
 tqdm==4.42.1
+Jinja2==3.0.0
```

Comment:
Why is Jinja2 required? I don't see any code that uses it

Reply:
save/load pretrained raises an error on huggingface_hub 0.24 and asks to install `Jinja2`

Reply:
alternatively we can bump minimum version for `huggingface_hub`